AdminCore.Services/Configuration/DependencyInjection.cs

```diff
@@ -18,8 +18,7 @@ public static class DependencyInjection
   {
     private static bool _registered;
 
-    public static void RegisterDependencyInjection(IServiceCollection serviceDescriptor = null,
-      params ServiceDescriptor[] serviceDescriptors)
+    public static void RegisterDependencyInjection(IServiceCollection serviceDescriptor = null)
```

Comment:
the idea here was to have two signatures call mostly the same implementation, rather than the parameters dictating which mappings are set. I'll grab some time with you and go over it.